components/activity_and_insights/insights/card/card.scss

```diff
@@ -0,0 +1,75 @@
+.Card.insights-card {
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    margin: 8px;
+    border-radius: 8px;
+    box-shadow: none;
```

Comment:
add hover state for card when mouse is within a card. Add these styles on hover:
```
box-shadow: 0px 6px 14px rgb(0 0 0 / 12%);
border-color: rgba(var(--center-channel-color-rgb),.16);
```